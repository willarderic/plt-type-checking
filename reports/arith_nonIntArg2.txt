TOKEN: LPAREN  LEXEME: (					TOKEN: LPAREN    LEXEME: (
TOKEN: SUB  LEXEME: -						TOKEN: SUB       LEXEME: -
ARITH_OP -> -							ARITH_OP -> -
BIN_OP -> ARITH_OP						BIN_OP -> ARITH_OP
TOKEN: INTCONST  LEXEME: 100					TOKEN: INTCONST  LEXEME: 100
CONST -> INTCONST						CONST -> INTCONST
EXPR -> CONST							EXPR -> CONST
TOKEN: STRCONST  LEXEME: "Hello"				TOKEN: STRCONST  LEXEME: "Hello"
CONST -> STRCONST						CONST -> STRCONST
EXPR -> CONST							EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR				ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
Line 1: Arg 2 must be integer					Line 1: Arg 2 must be integer
