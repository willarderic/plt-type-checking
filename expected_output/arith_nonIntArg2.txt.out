TOKEN: LPAREN    LEXEME: (
TOKEN: SUB       LEXEME: -
ARITH_OP -> -
BIN_OP -> ARITH_OP
TOKEN: INTCONST  LEXEME: 100
CONST -> INTCONST
EXPR -> CONST
TOKEN: STRCONST  LEXEME: "Hello"
CONST -> STRCONST
EXPR -> CONST
ARITHLOGIC_EXPR -> BIN_OP EXPR EXPR
Line 1: Arg 2 must be integer
